src/group_theory/schur_zassenhaus.lean

```diff
@@ -256,6 +254,7 @@ begin
   refine not_forall_not.mp (λ h3, _),
   haveI := by exactI
     schur_zassenhaus_induction.step7 hN (λ G' _ _ hG', by { apply ih _ hG', refl }) h3,
+  rw ← nat.card_eq_fintype_card at hN,
```

Comment:
The assumptions could be replaced by `finite G` and `nat.card G = n`, but that would be another PR to do it systematically for the whole file ...

Reply:
Yeah, there are a few group theory files that I'm planning on refactoring from `fintype` to `finite`.